nemo/lightning/pytorch/plugins/mixed_precision.py

```diff
@@ -225,7 +225,7 @@ def update_config_with_dtype_overrides(dtype_config, config):
         new_val = getattr(dtype_config, field.name)
```

Comment:
Small update: Change comment on line 223 from  `# If we overwrote a value, throw a warning.` to  `# If we overwrote a value, a debug message will be logged.`